### ISSUE TITLE
p4merge.bin: error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by phdoerfler** [NixOS/nixpkgs#73456]
## Issue description

p4merge is unable to start due to missing library.

### Steps to reproduce

```
$ nix-env -i p4v
$ p4merge
```

results in:

```
/nix/store/ncg8s6hk1r0drb1lx55s1p40a4lz6bv5-p4v-2017.3.1601999/bin/p4merge.bin: error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory
```

## Technical details

```
$ nix run nixpkgs.nix-info -c nix-info -m
- system: `"x86_64-linux"`
- host os: `Linux 4.19.82, NixOS, 19.09.1125.d628521d0b7 (Loris)`
- multi-user?: `yes`
- sandbox: `yes`
- version: `nix-env (Nix) 2.3`                                                                                                                                                                   - channels(philipp): `""`
- channels(root): `"nixos-19.09.1125.d628521d0b7"`
- nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
 ```

**Comment by veprbl** [comment]
cc @nathyong @nioncode

**Comment by veprbl** [comment]
Fixed by backporting #71301
